CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ Current Trunk
   - `module.getTableByIndex`
   - `module.getNumTables`
   - `binaryen.getTableInfo`
+- `BinaryenSetFunctionTable` and `module.setFunctionTable` have been removed
+  in favor of `BinaryenAddTable` and `module.addTable` respectively.
```

Comment:
This looks like a good change to me. I recently ran into issues with `BinaryenSetFunctionTable` yielding an "already exists" kind of error when a table had already been imported, and instead operating on a per-table basis seems like a desirable change.